packages/compass-query-bar/src/modules/change-filter.ts

```diff
@@ -0,0 +1,346 @@
+import {
```

Comment:
All these query change methods are the same as before, including tests, they are just extracted from the store